app/views/estimatedIncomeTax/DetailedIncomeTaxEstimateView.scala.html

```diff
@@ -33,148 +33,173 @@
-
-                        <p id="income-from-savings">
-                            @Messages("tai.estimatedIncome.savings.desc.totalIncomeEstimate", withPoundPrefix(MoneyPounds(viewModel.savings.map(_.income).sum,0)))
-                        </p>
-                        @if(DetailedIncomeTaxEstimateViewModel.containsHRS1orHRS2(viewModel.savings)) {
-                            <p id="higher-rate-para">
-                                @Messages("tai.estimatedIncome.savings.desc.higherRate", DetailedIncomeTaxEstimateViewModel.taxFreeSavingsIncome(viewModel.savings))
-                            </p>
+        @if(viewModel.dividends.nonEmpty){
+        <div class="section soft--top">
+            <h2 id="tax-on-dividend-income-text" class="govuk-heading-m heading-section govuk-!-padding-top-3">
+                @Messages("tai.estimatedIncome.detailedEstimate.dividendIncome.subHeading")</h2>
+            <p id="tax-on-dividend-income-desc" class="govuk-body">
+                @Messages("tai.estimatedIncome.dividend.para.desc",
+                MoneyPounds(viewModel.totalDividendIncome, 0).quantity,
+                MoneyPounds(viewModel.taxFreeDividendAllowance, 0).quantity)
+            </p>
+            @taxBandTable(tableClass = "employmentIncomeTaxDetails", taxBands = viewModel.dividends, taxRegion =
+            viewModel.taxRegion)
+        </div>
+        }
+
+        @if(viewModel.savings.nonEmpty){
+        <div class="section soft--top">
+            <h2 id="tax-on-savings-text" class="govuk-heading-m heading-section govuk-!-padding-top-3">
+                @Messages("tai.estimatedIncome.detailedEstimate.savingsInterest.subHeading")</h2>
+
+            <p id="income-from-savings" class="govuk-body">
+                @Messages("tai.estimatedIncome.savings.desc.totalIncomeEstimate",
+                withPoundPrefix(MoneyPounds(viewModel.savings.map(_.income).sum,0)))
+            </p>
+            @if(DetailedIncomeTaxEstimateViewModel.containsHRS1orHRS2(viewModel.savings)) {
+            <p id="higher-rate-para" class="govuk-body">
+                @Messages("tai.estimatedIncome.savings.desc.higherRate",
+                DetailedIncomeTaxEstimateViewModel.taxFreeSavingsIncome(viewModel.savings))
+            </p>
+            }
+            @taxBandTable(tableClass = "employmentIncomeTaxDetails", taxBands = viewModel.savings.toList, taxRegion =
+            viewModel.taxRegion)
+
+        </div>
+        }
+
+        @if(viewModel.additionalTaxTable.nonEmpty){
+        <div>
+            <h3 id="additionalTaxTable-heading" class="govuk-heading-m heading-section govuk-!-padding-top-3">
+                @Messages("tai.estimatedIncome.additionalTax.title")</h3>
+            <p id="additionalTaxTable-desc" class="govuk-body">@Messages("tai.estimatedIncome.additionalTax.desc")</p>
+            <table id="additionalTaxTable" class="govuk-table">
```

Comment:
table is missing a caption 
will fail accessibility checks...
https://design-system.service.gov.uk/components/table/

Reply:
perfect thanks!